### ISSUE TITLE
#a11y Tab sequence

### ISSUE THREAD

**Issue by joedolson** [obenland/shiny-updates#72]
When updating themes, tabbing through the options should happen in the same visual order as the appearance. Right now, the tab order is Details > Customize > Update Now; but should be Details > Update Now > Customize. 

I think this would also better match user needs, since if a theme needs an update, they most likely would want to apply the update before customizing, so that would be the priority task.
